app/_how-tos/configure-oidc-with-jwt-auth.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Configure OpenID Connect with JWT authentication
+content_type: how_to
+
+related_resources:
+  - text: Authentication in {{site.base_gateway}}
+    url: /gateway/authentication/
+  - text: OpenID Connect authentication flows and grants
+    url: /plugins/openid-connect/#authentication
+  - text: JWT auth flow
+    url: /plugins/openid-connect/#jwt-access-token-authentication-flow
+
+plugins:
+  - openid-connect
+
+entities:
+  - route
+  - service
+
+products:
+  - gateway
+
+works_on:
+  - on-prem
+  - konnect
+
+min_version:
+  gateway: '3.4'
+
+tools:
+  - deck
+
+prereqs:
+  entities:
+    services:
+      - example-service
+    routes:
+      - example-route
+  inline:
+    - title: Set up Keycloak
+      include_content: prereqs/auth/oidc/keycloak-password
+      icon_url: /assets/icons/keycloak.svg
+
+tags:
+  - authentication
+  - openid-connect
+
+description: Set up OpenID Connect with JSON Web Token (JWT) auth, which uses a bearer token for authentication with the IdP.
+
+tldr:
+  q: How do I retrieve a JWT token and use it to authenticate with my IdP?
+  a: Using the OpenID Connect plugin, set up the [JWT auth workflow](/plugins/openid-connect/#jwt-access-token-authentication-flow) to connect to an identity provider (IdP) and use the validated token to access the upstream service.
+
+cleanup:
+  inline:
+    - title: Clean up Konnect environment
+      include_content: cleanup/platform/konnect
+      icon_url: /assets/icons/gateway.svg
+    - title: Destroy the {{site.base_gateway}} container
+      include_content: cleanup/products/gateway
+      icon_url: /assets/icons/gateway.svg
+
+search_aliases:
+  - oidc
+---
+
+## 1. Enable the OpenID Connect plugin with JWT authentication
+
+Stateless JWT Access Token authentication is named `bearer` in the OpenID Connect plugin.
+
+Using the Keycloak and {{site.base_gateway}} configuration from the [prerequisites](#prerequisites), 
+set up an instance of the OpenID Connect plugin with bearer authentication. 
+We're also enabling the password grant so that you can test retrieving the bearer auth token.
```

Comment:
```suggestion
Using the Keycloak and {{site.base_gateway}} configuration from the [prerequisites](#prerequisites), 
set up an instance of the OpenID Connect plugin with bearer authentication. The stateless JWT Access Token authentication is named `bearer` in the OpenID Connect plugin.

We're also enabling the password grant so that you can test retrieving the bearer auth token. 
```